src/renderer/components/_atoms/highlighted-textarea/index.tsx

```
@@ -0,0 +1,145 @@
+      <div className='[&::-webkit-text-size-adjust]:none relative h-full w-full' aria-label={props['aria-label']}>
+        <div
+          className={cn(
+            'pointer-events-none absolute -z-10 w-full overflow-y-scroll [&::-webkit-scrollbar]:hidden',
+            'left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2',
+            highlightClassName,
+          )}
+          ref={highlightDivRef}
+        >
+          <div
+            className={cn('w-full whitespace-pre-wrap break-words text-transparent')}
+            dangerouslySetInnerHTML={{ __html: formattedVariableValue }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)